ci/pipelines/deploy.yml

```diff
@@ -304,15 +304,22 @@ jobs:
     serial_groups: [directdebit-frontend-stg]
     plan:
       - get: omnibus
-      - get: directdebit-frontend
+      - get: git-directdebit-frontend-pre-release
         trigger: true
+      - task: download-directdebit-frontend-artifact
+        file: omnibus/ci/tasks/download-deployment-artifact.yml
```

Comment:
I thought the github-release resource would download artifacts for you:
https://github.com/concourse/github-release-resource#in-fetch-assets-from-a-release

Reply:
yeah it does and we're taking it from there, the name is misleading, it would be better named something like `extract-artefact` since that's what its doing.. naming things is hard... I'll correct it.
https://github.com/alphagov/pay-omnibus/blob/develop/ci/tasks/download-deployment-artifact.yml